com.civicrm.hrjobroles/views/include/add_new_role.html

```diff
@@ -85,63 +85,104 @@
+                        <ui-select-choices prevent-animations repeat="department.value.value as (key,department) in jobroles.DepartmentsData | filter: { value: { title: $select.search }}">
+                          <div ng-bind-html="department.value.title | highlight: $select.search"></div>
+                        </ui-select-choices>
+                      </ui-select>
+                    </div>
+                    <a href="/civicrm/admin/options?gid=112&reset=1"
+                       class="crm-option-edit-link medium-popup crm-hover-button"
+                       target="_blank" title="Edit Task Types"
+                       data-option-edit-path="civicrm/admin/options?gid=112&reset=1">
+                      <i class="crm-i fa-wrench"></i>
+                    </a>
+                  </div>
                 </div>
               </div>
               <div class="form-group">
                 <label for="newLevel" class="col-xs-4 control-label">Level:</label>
                 <div class="col-xs-6">
-                  <ui-select
-                    prevent-animations
-                    ng-model="jobroles.editData['new_role_id']['level']"
-                    theme="civihr-ui-select"
-                    title="Choose a Level">
-                    <ui-select-match prevent-animations allow-clear placeholder="Select a level">{{$select.selected.value.title}}</ui-select-match>
-                    <ui-select-choices prevent-animations repeat="level.value.value as (key,level) in jobroles.LevelsData | filter: { value: { title: $select.search }}">
-                      <div ng-bind-html="level.value.title | highlight: $select.search"></div>
-                    </ui-select-choices>
-                  </ui-select>
+                  <div class="form-inline">
+                    <div class="crm_custom-select">
+                      <ui-select
+                        prevent-animations
+                        ng-model="jobroles.editData['new_role_id']['level']"
+                        theme="civihr-ui-select"
+                        title="Choose a Level">
+                        <ui-select-match prevent-animations allow-clear placeholder="Select a level">{{$select.selected.value.title}}</ui-select-match>
+                        <ui-select-choices prevent-animations repeat="level.value.value as (key,level) in jobroles.LevelsData | filter: { value: { title: $select.search }}">
+                          <div ng-bind-html="level.value.title | highlight: $select.search"></div>
+                        </ui-select-choices>
+                      </ui-select>
+                    </div>
+                    <a href="/civicrm/admin/options?gid=111&reset=1"
+                       class="crm-option-edit-link medium-popup crm-hover-button"
+                       target="_blank" title="Edit Task Types"
+                       data-option-edit-path="civicrm/admin/options?gid=111&reset=1">
```

Comment:
Why do we use `gid=111`? Looks like we request data by fixed ID. Are there any other places in the code using the same technique? If yes, please refer to the exact line of existing code. This is absolutely essential to describe this in the Technical Description section.

Reply:
I actually worked with a [document](https://docs.google.com/spreadsheets/d/1cXYoRoBv0XV5UXruxw33y_OR4CiyIwLRKjbh4i1yEGo/edit#gid=0) in order to get the url to render. I will check if there is a better way to achieve this.

Reply:
@ajesamson Maybe this is an expected approach? Please look at other examples in the existing code and provide a reference to them at first.